### PR TITLE
VM: Implement DIV, MOD opcode

### LINKED ISSUE #224: VM: implement DIV , MOD opcode

### PR DESCRIPTION
**Pull request by dauTT**
Closes #224

### CODE REVIEW

File: pkg/vm/vm_ops_maths_test.go [hunk 1]
```diff
@@ -117,3 +117,63 @@ func TestSubOp(t *testing.T) {
...
+	// Stack should have one item
+	assert.Equal(t, 1, ctx.Estack.Len())
+
+	item, err := ctx.Estack.PopInt()
+	if err != nil {
+		t.Fail()
+	}
+
+	assert.Equal(t, int64(2), item.Value().Int64())
+
```
Comment (decentralisedkev) [inline pkg/vm/vm_ops_maths_test.go]: remove empty line

File: pkg/vm/vm_ops_maths_test.go [hunk 2]
```diff
@@ -117,3 +117,63 @@ func TestSubOp(t *testing.T) {
...
+	// Stack should have one item
+	assert.Equal(t, 1, ctx.Estack.Len())
+
+	item, err := ctx.Estack.PopInt()
+	if err != nil {
+		t.Fail()
+	}
+
+	assert.Equal(t, int64(3), item.Value().Int64())
+
```
Comment (decentralisedkev) [inline pkg/vm/vm_ops_maths_test.go]: remove empty line. Except that LGTM

File: pkg/vm/vm_ops_maths_test.go (hunk 1 above)
Comment (dauTT) [reply pkg/vm/vm_ops_maths_test.go]: :+1:

File: pkg/vm/vm_ops_maths_test.go (hunk 2 above)
Comment (dauTT) [reply pkg/vm/vm_ops_maths_test.go]: :+1:

**Comment by hal0x2328** [comment]
100 points dauTT

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/630c74684cc6dad330397c63e8c7aa6403c043cc1f241638cff87e3b97a40716

**Pull request merged**